### PR TITLE
Rewrite the Readme for EFCore

### PR DESCRIPTION
**Pull request by EugeniyBurmistrov**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -4,81 +4,40 @@
...
 
-# How to connect different data models to several databases within a single application
-
-> **Note**  
-> The description of this example is currently under construction and may not match the code in the example. We are currently working to provide you with up-to-date content.
-
-## Scenario
+# How to connect different data models to several databases within a single application with Entity Framework Core
 
 This example demonstrates how to create custom XAF modules with custom business objects and logic that would work with separate databases. These modules do not depend on each other and thus can be reused in other applications as a whole.
```
Comment (albertov05) [inline Readme.md]: 1. `This example demonstrates how to create custom XAF modules with custom business objects and logic that would work with separate databases.`

Can you change "that would work with separate databases" to "that work with separate databases"?

2. `These modules do not depend on each other and thus can be reused in other applications as a whole.`

By "as a whole" do you mean "independently"?

File: Readme.md [hunk 2]
```diff
@@ -4,81 +4,40 @@
...
-
-**3.** Add service _ModuleInfo_ classes into these modules as shown in the _ClassLibraryN/ModuleInfoN.xx_ files (only if you want to use [ModuleInfo](https://docs.devexpress.com/eXpressAppFramework/112795/deployment/update-application-and-database-versions-using-the-module-info-table/) to control migrations).
-
-**4.** In _YourModuleName/Module.xx_ files, override the _Setup(XafApplication application)_ methods of the ModuleBase descendants to handle the _CreateCustomObjectSpaceProvider_ event of the XafApplication class as shown in the _ClassLibraryX/XafModuleN.xx_ files.
 
-**5.** Build the solution, invoke the [_Module Designer_](https://docs.devexpress.com/eXpressAppFramework/112828/installation-upgrade-version-history/visual-studio-integration/module-designer) for the platform-agnostic module (_YourSolutionName.Module/Module.xx_), and drag the created custom modules from the Toolbox:
+The [EFCore](./CS/EFCore/) example solution illustrates the following documentation article: [How to: Use Multiple Data Models Connected to Different Databases in Entity Framework Core](https://docs.devexpress.com/eXpressAppFramework/404322).
```
Comment (albertov05) [inline Readme.md]:
```suggestion
This [EFCore](./CS/EFCore/) example solution is based on the following tutorial: [How to: Use Multiple Data Models Connected to Different Databases in Entity Framework Core](https://docs.devexpress.com/eXpressAppFramework/404322).
```

File: Readme.md (hunk 1 above)
Comment (EugeniyBurmistrov) [reply Readme.md]: I applied both suggestions, thank you!

Comment (albertov05) [reply Readme.md]: Thank you!

**Pull request merged**